content/docs/welcome-guide/requirements.md

```diff
@@ -11,7 +11,7 @@ prerequisites and configuration](#software-prerequisites)
 section, each prerequisite is listed along with any specific configuration steps
 required.
 
-## Hardware requirements
+# Hardware requirements
```

Comment:
It goes against the doc style to use an H1 header for these subsections. However, looking at the deploy preview, I think I see why you did this: the most useful links remain in the right-hand ToC after adding the new sections.

Instead of using H1 headers though, I'd rather keep them at H2, while keeping **Microsoft Windows** and the remaining OS sections at H2.
```suggestion
## Hardware requirements
```